test/index.js

```diff
@@ -732,4 +825,53 @@ lab.experiment('Normal setup', function (done) {
         });
     });
 
+    lab.test('policy can be added programmatically', function (done) {
+        server.plugins.mrhorse.addPolicy('yetAnotherPolicy', (request, reply, callback) => callback(null, true));
+
+        Code.expect(server.plugins.mrhorse.hasPolicy('yetAnotherPolicy')).to.equal(true);
+        done();
+    });
+
+    lab.test('hasPolicy returns false on non-existent policies', function (done) {
+        Code.expect(server.plugins.mrhorse.hasPolicy('thisPolicyAbsolutelyDoesNotExist')).to.equal(false);
+        done();
+    });
```

Comment:
I'd also like to see a test where the programatically added policy is attached to a route and tested.